src/Hadrian/Haskell/Cabal/Configured.hs

```diff
@@ -0,0 +1,47 @@
+module Hadrian.Haskell.Cabal.Configured where
+
+import Development.Shake.Classes
+import Hadrian.Package.Type
+import GHC.Generics
+
+data ConfiguredCabal = ConfiguredCabal
```

Comment:
The name `ConfiguredCabal` is a bit confusing to me. Can we keep the old `PackageData`? I think it's much more clear and people familiar with the old build system will immediately understand what it means.

Reply:
OK. Fixed in an upcoming commit.